From ceae8c623fdc323be12014f50041fbddee4c5ed5 Mon Sep 17 00:00:00 2001
From: Bryan O'Donoghue <bryan.odonoghue@linaro.org>
Date: Sat, 7 Sep 2019 16:13:21 +0100
Subject: [PATCH 12/13] pico-imx6ul: Define the name of the FIT bootscript
 subimage

Defines BOOTENV_FIT_BOOTSCRIPT_SUBIMAGE to ":bootscr" so that the source
command can reference the appropriate sub-image when doing distroboot using
a FIT image.

Signed-off-by: Bryan O'Donoghue <bryan.odonoghue@linaro.org>
---
 include/configs/pico-imx6ul.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/include/configs/pico-imx6ul.h b/include/configs/pico-imx6ul.h
index cd051bf263..9cd7be0e8b 100644
--- a/include/configs/pico-imx6ul.h
+++ b/include/configs/pico-imx6ul.h
@@ -106,6 +106,11 @@
 	func(PXE, pxe, na) \
 	func(DHCP, dhcp, na)
 
+/* Define the name of the FIT subimage for the bootscript */
+#if defined(CONFIG_FIT)
+#define BOOTENV_FIT_BOOTSCRIPT_SUBIMAGE ":bootscr"
+#endif
+
 #include <config_distro_bootcmd.h>
 
 #define CONFIG_SYS_MEMTEST_START	0x80000000
-- 
2.22.0

